### DISCUSSION TITLE
Tree support wall loops

### DISCUSSION THREAD

**Discussion by AL37** [OrcaSlicer/OrcaSlicer#2678] [Dev]
I can search for Tree Support Wall Loops, but when I go to the Tree Support section it is not there.

**Comment by LJKraus** [comment]
Wall Loops can be changed for the Tree Slim, Strong, and Hybrid styles.  No wall loop options for Default or Organic styles.

**Reply by RadTechDad** [reply]
The option should probably still be there or at least disabled so that users can see that it's not missing.

**Reply by Noisyfox** [reply]
Yeah there is an issue to improve this https://github.com/SoftFever/OrcaSlicer/issues/2170
